src/sentry/static/sentry/app/components/charts/lineChart.jsx

```diff
@@ -22,6 +22,10 @@ export default class LineChart extends React.Component {
             ...options,
             name: seriesName,
             data: dataArray || data.map(({value, name}) => [name, value]),
+            color: '#948BCF',
+            areaStyle: {
+              color: '#C4BFE9',
+            },
```

Comment:
You might want to make these props that are passed in as to not impact dashboards.

Reply:
Will change the usage of lineChart to areaChart 👍